backend/config.py

```
@@ -4,6 +4,19 @@
 class Config:
     # Flask configuration
     SECRET_KEY = os.environ.get("SECRET_KEY", "default_secret_key")
-    SQLALCHEMY_DATABASE_URI = "sqlite:///instance/site.db"
+    SQLALCHEMY_DATABASE_URI = os.environ.get(
+        "DATABASE_URL", "sqlite:///instance/site.db"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)